src/styles/HomePage.module.css

```diff
@@ -0,0 +1,231 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+/* Books display section */
+.ul {
+  border-bottom: 1px solid #e8e8e8;
+  padding: 0 0 2.5rem;
+  margin: 2.313rem 6.25rem 1.813rem;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
```

Comment:
- [ ]  this line also is repeated several times, will you refactor in order to make your code DRY